Controller/ContentTypeController.php

```diff
@@ -127,6 +140,8 @@ public function viewContentTypeGroupAction($contentTypeGroupId)
             )->createView();
 
             $canDeleteById[$contentTypeId] = $canDelete && !$this->contentTypeService->isContentTypeUsed($contentType);
+
+            $languageById[$contentTypeId] = $this->getVerifiedContentTypeLanguageCode($contentType);
```

Comment:
Think we need to simplify this. Because of the extra helper functions here it is basically loading all languages over and over again for each content type.

So what do we need to extract here? Afaik you are only interested in which languages content type is already translated to right? So this can then simply extract this from `array_keys($contentType->names)` if we can live with referring to language by language code.

If not, why do you need id? :P